corehq/apps/ivr/migrations/0002_call_app_id.py

```diff
@@ -1,32 +1,34 @@
+from collections import defaultdict
 from django.db import migrations, models, transaction
 
 from corehq.apps.app_manager.dbaccessors import get_apps_in_domain
 from corehq.util.django_migrations import skip_on_fresh_install
 
 
-def _update_model(model, domain, app_id_by_form_unique_id):
+def _update_model(model, domain, domain_forms):
     if model.form_unique_id is None:
         return None
 
-    if model.form_unique_id not in app_id_by_form_unique_id:
+    if domain not in domain_forms:
+        domain_forms[domain] = defaultdict(dict)
```

Comment:
It looks like the thing stored in `domain_forms['domain']` is a regular dict mapping form unique ID to app ID

`domain_forms` itself is a dict of dicts, but I don't think it needs to be a `defaultdict`